src/lib/swapper.ts

```diff
@@ -217,6 +240,17 @@ export async function getSwapperDBalances(options: { network: string }): Promise
     return response;
 }
 
+export async function getSwapperDSwaps(options: { network: string }): Promise<SwapsResponse> {
+    let response: SwapsResponse;
+    try {
+        response = (await axios.get(`${API}/swaps?network=${options.network}`)).data;
+    } catch (error) {
+        throw updateError(`${errors.UnableToRetrieveSwaps}: ${error.message || error}`, error);
+    }
+
+    return response;
```

Comment:
Once SwapperDsupports returning back an origin, we will want to filter out swaps that weren't created by RenEx.